src/njoy21/input/LEAPR/Card1.hpp

```diff
@@ -0,0 +1,16 @@
+class Card1 {
+public:
+  #include "njoy21/input/LEAPR/Card1/Nout.hpp"
+  Argument < Nout > nout;
```

Comment:
There is a `verify` function that is not dependent on `nin`. See [https://github.com/njoy/NJOY21/blob/master/src/njoy21/input/argument/common/Nout.hpp#L21]()